kubesae/info.py

```diff
@@ -1,24 +1,45 @@
 import invoke
 import yaml
 
-@invoke.task(default=True)
-def get_ansible_vars(c, var=None):
-    """Inspect ansible variables
+@invoke.task("print_ansible_vars")
+def print_ansible_vars(c, var=None, yaml=None, pty=True, hide=False):
+    """A command to inspect any ansible varible by environment. If no variable is specified then it will
+    print out the current k8s environment variables.
 
-    Displays Ansible variables in the specified environment. If no variable is specified
-    then it will print out the current k8s environment variables.
+    Args:
+        c (invoke.Context): [description]
+        var (string, optional): The ansible variable you want to expose. Defaults to None.
+        yaml (string, optional): An ansible path. Defaults to None.
+        pty (bool, optional): [description]. Defaults to True.
+        hide (bool, optional): [description]. Defaults to False.
 
-    Params:
-        var: A variable available to a host when called.
-
-    Usage: inv <ENVIRONMENT> info
-           inv <ENVIRONMENT> info --var=<ANSIBLE_VAR>
+    Returns:
+        invoke.Result
+    
+    Usage:
+        $ inv staging info.print_ansible_vars
+            Prints all of the environment variables values defined in the 
+            "k8s_environment_variables" dictionary located at 
+            <PROJECT_ROOT>/deploy/host_vars/staging.yml.
 
+        $ inv staging info.print_ansible_vars --var=foo_bar_baz
+            Prints the "foo_bar_baz" value defined in the "foo_bar_baz"
+            variable located at <PROJECT_ROOT>/deploy/host_vars/staging.yml
+        
+        $ inv staging info.print_ansible_vars --var=foo_bar_baz --yaml="@group_vars/all.yaml"
+            Prints the "foo_bar_baz" value defined in the "foo_bar_baz"
+            variable located at <PROJECT_ROOT>/deploy/group_vars/all.yml
     """
-    if not var:
-        var = "k8s_environment_variables"
+    yaml_file = f"@host_vars/{c.config.env}.yml"
```

Comment:
Can we make this something like
```
f"@host_vars/{c.config.env}.yml" if os.path.exists(f"deploy/host_vars/{c.config.env}.yml") else f"@host_vars/{c.config.env}.yaml"
```
... like we do [here](https://github.com/caktus/invoke-kubesae/blob/master/kubesae/ansible/deploy.py#L52)?